src/sys/socket/mod.rs

```diff
@@ -988,6 +1020,8 @@ impl<'a> ControlMessage<'a> {
               target_os = "netbsd", target_os = "freebsd",
               target_os = "android", target_os = "ios",))]
             ControlMessage::Ipv6PacketInfo(_) => libc::IPPROTO_IPV6,
+            #[cfg(target_os = "linux")]
+            ControlMessage::RxqOvfl(_) => libc::SO_RXQ_OVFL,
```

Comment:
Could you double-check this part?

Reply:
I updated the condition to include `android`, `fuchsia`, `linux`. CI seems running ok.

Reply:
I mean that `SO_RXQ_OVFL` doesn't look like a valid value for `cmsg_level`.  Is that really correct?

Reply:
Oh right. updated to `libc::SOL_SOCKET`. thanks!